Maestro/subscriptions.json

```diff
@@ -1474,40 +1396,6 @@
           "CommitsToMirror": "<trigger-commitlist>"
         }
       }
-    },
-    // Automate opening PRs to merge winforms-designer release/16.10p4 to release/16.11p1
```

Comment:
azdo repository is disabled
https://dev.azure.com/dnceng/internal/_git/dotnet-winforms-designer

Reply:
This should stay. This repo builds from GH.